app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -10,14 +10,19 @@ class SessionsController < ActionController::Base
     end
 
     def new
-      authenticate if sanitized_shop_name.present?
+      if sanitized_shop_name.present?
+        Rails.logger.debug("[ShopifyApp::SessionsController] Sanitized shop name present. Authenticating...")
+        authenticate
+      end
     end
 
     def create
+      Rails.logger.debug("[ShopifyApp::SessionsController] Authenticating...")
```

Comment:
Should this say "Create"?

Reply:
@rezaansyed Do you mind adding `Create` to this log message on your upcoming PR? Thanks!